backend/src/services/resource-cleanup/resource-cleanup-queue.ts

```diff
@@ -25,6 +34,9 @@ export const dailyResourceCleanUpQueueServiceFactory = ({
     await auditLogDAL.pruneAuditLog();
     await identityAccessTokenDAL.removeExpiredTokens();
     await secretSharingDAL.pruneExpiredSharedSecrets();
+    await snapshotDAL.pruneExcessSnapshots();
```

Comment:
snapshot pruning should complete before any secretversion and secretfolderversion cleanup happens